app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RecipesController <ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @recipe_cost = Recipe.find(params[:id]).total_cost
```

Comment:
I would just do `@recipe.total_cost` in your view. That would not be considered and MVC infraction because the method that does that calculation lives in the Model where it should. 

The other thing that isn't ideal about this is you are reaching in to the database twice. Once for your `find` on line 3 and oncee for `find` line 4. You don't need to find the same recipe twice within the same method.